### PR TITLE
Fix PVS-Studio warnings, part 3 + some other fixes

### PR DESCRIPTION
**Pull request by cher-nov**
This is the last PR from my series of cosmetix changes.

### CODE REVIEW

File: libspawner/src/win32/system_pipe.cpp [hunk 1]
```diff
@@ -98,24 +98,24 @@ size_t system_pipe::read(char* bytes, size_t count) const {
     if (!is_readable())
         return 0;
 
-    size_t readed;
```
Comment (ViktorPegy) [inline libspawner/src/win32/system_pipe.cpp]: read_count

File: libspawner/src/win32/system_pipe.cpp [hunk 2]
```diff
@@ -98,24 +98,24 @@ size_t system_pipe::read(char* bytes, size_t count) const {
...
 
-    return readed;
+    return num_read;
 }
 
 size_t system_pipe::write(const char* bytes, size_t count) const {
     if (!is_writable())
         return 0;
 
-    size_t writed;
```
Comment (ViktorPegy) [inline libspawner/src/win32/system_pipe.cpp]: write_count

File: libspawner/src/win32/system_pipe.cpp (hunk 1 above)
Comment (cher-nov) [inline libspawner/src/win32/system_pipe.cpp]: `num_read` => "__num__ber of bytes __read__"

File: libspawner/src/win32/system_pipe.cpp (hunk 2 above)
Comment (cher-nov) [inline libspawner/src/win32/system_pipe.cpp]: `num_written` => "**num**ber of bytes **written**"

File: libspawner/src/win32/system_pipe.cpp (hunk 1 above)
Comment (klenin) [inline libspawner/src/win32/system_pipe.cpp]: Make it `bytes_read` and `bytes_written`

Comment (cher-nov) [inline libspawner/src/win32/system_pipe.cpp]: @klenin Done.

**Pull request merged**